lab-glen/README.md

```diff
@@ -0,0 +1,24 @@
+###### Welcome to SACK CHAT!
+
+
+### How to use
+
+1. In command terminal type  ```telnet localhost 3000```
+2. You will be logged in
+3. Type a command to chat!
+
+
+### Commands
+/nick - This changes your nickname. Do this first.
```

Comment:
Will want to add more next time such as small paragraph about the project, how to use it for people that have never used this application before.